<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingWidget</class>
 <widget class="QWidget" name="TrainingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCustomPlot" name="bigChart" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="gameLayout_3"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="setLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_analyze">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Analyze</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QSpinBox" name="sb_szBatches">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="singleStep">
          <number>2</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbl_szTraining">
         <property name="text">
          <string>Events for Training</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbl_szValidation">
         <property name="text">
          <string>Events for Test</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QComboBox" name="cb_annMode">
         <property name="currentText">
          <string>FC(Softmax)</string>
         </property>
         <item>
          <property name="text">
           <string>FC(Softmax)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FC(64 LReLU, Softmax)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FC(8x8, LReLU, Softmax)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CNN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>I</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>J</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lbl_szBatches">
         <property name="text">
          <string>Batch size</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lbl_drawMode">
         <property name="text">
          <string>Draw</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="sb_szEpochs">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lbl_annMode">
         <property name="text">
          <string>Topology</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="sb_szTraining">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_loadData">
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Load Data</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="cb_drawMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentText">
          <string>Training/Test</string>
         </property>
         <item>
          <property name="text">
           <string>Training/Test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Theta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Momentum</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lbl_szEpochs">
         <property name="text">
          <string>Quantity of Epochs</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="sb_szValidation">
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="btn_loadNet">
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btn_saveNet">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exportAccuracy">
       <property name="text">
        <string>Export Accuracy Graph (.nnp)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="smallChart" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>50</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pro_Epoch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pro_Training">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_loadData</tabstop>
  <tabstop>btn_analyze</tabstop>
  <tabstop>sb_szTraining</tabstop>
  <tabstop>sb_szValidation</tabstop>
  <tabstop>sb_szEpochs</tabstop>
  <tabstop>sb_szBatches</tabstop>
  <tabstop>cb_annMode</tabstop>
  <tabstop>cb_drawMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
